### ISSUE TITLE
Search Lockbox Entries input box is missing a label

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-lockbox/lockbox-extension#556]
The search input should have a label to describe its purpose to screenreader users.

Example: `<input type="search" label="Search Entries"` or `"Search Lockbox Entries"`>
